Floofbot/Floofbot.csproj

```diff
@@ -7,7 +7,7 @@
 
   <ItemGroup>
     <FrameworkReference Include="Microsoft.AspNetCore.App" />
-    <PackageReference Include="Discord.Addons.Interactive" Version="1.0.1" />
+    <ProjectReference Include="..\Discord.Addons.Interactive\Discord.Addons.Interactive.csproj" />
```

Comment:
Change to / for Linux pls. Pretty sure Windows supports forward slashes for these paths, too.

Reply:
done